### PR TITLE
repair crash when opening non-PDF files

### LINKED ISSUE #31: Crashes when opening non-PDF files

### PR DESCRIPTION
**Pull request by Ra-V2**
Fixes #31

### CODE REVIEW

File: application/MainWindow.cpp [hunk 1]
```diff
@@ -207,7 +207,14 @@ MainWindow::MessageReceived(BMessage* message)
         	int32 page = 0;
         	message->FindInt32("page", &page);
         	auto type = _FileType(file);
-    		_OpenFile(file, type, password, page);
+        	if (type == "djvu" || type == "pdf" || type == "xps") {
+    			_OpenFile(file, type, password, page);
+        	}
+        	else {
+        		BAlert *alert = new BAlert("Error", "Wrong file format.", "Ok", NULL, NULL,
```
Comment (humdingerb) [inline application/MainWindow.cpp]: In Haiku we normally use all uppercase "OK".

File: application/MainWindow.cpp [hunk 2]
```diff
@@ -258,7 +258,7 @@ MainWindow::_FileType(BString const& file)
 		if (file.IFindLast("xps") != B_ERROR)
 			type = "xps";
 	}
-
+
```
Comment (humdingerb) [inline application/MainWindow.cpp]: Remove stray tab, please.

**Comment by owenca** [comment]
@Ra-V2 Another student chose to fix this bug on GCI website before you. Please pick a different issue and **check with us first**.

**Comment by humdingerb** [comment]
Sorry about that. I wasn't a mentor for that task and not aware of it...

**Comment by owenca** [comment]
No problem! Maybe we should ask students to leave a note here in addition to on GCI website.

**Comment by humdingerb** [comment]
Wouldn't have helped in this case, I think. I recognized Ra-V2 as GCI student, but that didn't make me aware of Emily's efforts on the same issue.
No matter. We lernt: working on something without claim on the GCI task is risky...

**Pull request closed without merging**